test/e2e/node/security_context.go

```diff
@@ -77,6 +79,42 @@ var _ = SIGDescribe("Security Context", func() {
 		e2eoutput.TestContainerOutput(f, "pod.Spec.SecurityContext.SupplementalGroups", pod, 0, groups)
 	})
 
+	ginkgo.When("if the container's primary UID belongs to some groups in the image [LinuxOnly]", func() {
+		ginkgo.It("should add pod.Spec.SecurityContext.SupplementalGroups to them [LinuxOnly] in resultant supplementary groups for the container processes", func() {
+			// using the image used in cri-tools
+			// ref: https://github.com/kubernetes-sigs/cri-tools/blob/master/images/image-predefined-group/Dockerfile
+			pod := scTestPod(false, false)
+			pod.Spec.Containers[0].Image = "gcr.io/k8s-staging-cri-tools/test-image-predefined-group:latest"
```

Comment:
we also avoid use of `latest` so any future changes to this image will have no chance of breaking this test on previous release branches

Reply:
OK. I will update this later.

Reply:
fixed in d92e429

- the image is defined in https://github.com/kubernetes/kubernetes/pull/113168
- postsubmit job for the image is defined in https://github.com/kubernetes/test-infra/pull/27772

Reply:
https://github.com/kubernetes/kubernetes/pull/113168#issuecomment-1284779342 suggested me to re-use agnhost E2E image for this test case instead of introducing a new E2E image. So, I opened https://github.com/kubernetes/kubernetes/pull/113185 which adds a test user and a group in the image.

Updated to use agnhost image in f051d66